config.json

```diff
@@ -1321,7 +1332,12 @@
         "name": "Two Bucket",
         "uuid": "0bc6b478-40a8-47ab-889b-c403b922f7e5",
         "practices": [],
-        "prerequisites": [],
+        "prerequisites": [
+          "array-analysis",
+          "classes",
+          "conditionals",
+          "functions"
```

Comment:
For this one I would go by the current example solution. It was recently updated and is rather readable. There I see classes, arithmetic-operators, errors, conditionals, while-loops and objects (for the return value).